Source/Core/VideoCommon/VideoBackendBase.cpp

```diff
@@ -48,8 +33,7 @@ void VideoBackend::PopulateList()
 	// OGL > D3D11 > SW
 	g_available_video_backends.push_back(backends[0] = new OGL::VideoBackend);
 #ifdef _WIN32
-	if (IsGteVista())
-		g_available_video_backends.push_back(backends[1] = new DX11::VideoBackend);
+	g_available_video_backends.push_back(backends[1] = new DX11::VideoBackend);
 #endif
 	g_available_video_backends.push_back(backends[3] = new SW::VideoSoftware);
```

Comment:
backends[2] ~~is~~ was the DirectX 9 backend.

Nobody bothered to shrink the array when it was removed. No point really, it just gets filled with a null value and things that iterate over the list are designed to cope with gaps.